apps/vpnmanager/src/components/Statistics/Statistics.tsx

```diff
@@ -0,0 +1,313 @@
+import React, { useEffect, useRef, useState } from "react";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  TextField,
+  Grid,
+  Button,
+  TablePagination,
+} from "@mui/material";
+import { Section } from "@commons-ui/core";
+import { useRouter } from "next/router";
+
+import { fetchJson, formatBytes } from "@/vpnmanager/utils";
+import { Link } from "@commons-ui/next";
+import { format, startOfYesterday } from "date-fns";
+
+export interface Data {
+  ID: number;
+  userId: string;
+  usage: number;
+  date: string;
+  cumulativeData: number;
+  email: string;
+  createdAt: string;
+}
+
+interface Props {
+  data: Data[];
+}
+
+const Statistics: React.FC<Props> = ({ data: result }) => {
+  const router = useRouter();
+  const yesterday = startOfYesterday();
+  const [filters, setFilters] = useState({
+    ID: router.query.ID || "",
+    userId: router.query.userId || "",
+    email: router.query.email || "",
```

Comment:
Do we need all three? Remember this is a security app so we shouldn't show anything we don't **need** to show.

My feeling is we only need to show email address to the security team both Sqlite ID and Outline ID shouldn't be visible anywhere.